tests/yam/agama/boot_agama.pm

```diff
@@ -63,7 +76,6 @@ sub prepare_boot_params {
 
 sub run {
     my $self = shift;
-
```

Comment:
is typical to leave a blank line after the unpacking in Perl, you can change "here" by "above" if that helps in the comments.